Makefile

```diff
@@ -0,0 +1,113 @@
+# Package configuration
+PROJECT := gitbase-playground
+COMMANDS := cmd/gitbase-playground
+DOCKER_ORG := src-d
+DEPENDENCIES := \
+	github.com/golang/dep/cmd/dep \
+	github.com/jteeuwen/go-bindata \
+	github.com/golang/lint/golint
+GO_LINTABLE_PACKAGES := $(shell go list ./... | grep -v '/vendor/')
+GO_BUILD_ENV := CGO_ENABLED=0
+
+# Tools
+GODEP := dep
+GOLINT := golint
+GOVET := go vet
+BINDATA := go-bindata
+DIFF := diff
+GITADD := git add -A
+
+# Default rule
+all:
+
+# Including ci Makefile
+CI_REPOSITORY ?= https://github.com/dpordomingo/ci.git
+CI_BRANCH ?= v1
+CI_PATH ?= $(shell pwd)/.ci
+MAKEFILE := $(CI_PATH)/Makefile.main
+$(MAKEFILE):
+	@git clone --quiet --depth 1 -b $(CI_BRANCH) $(CI_REPOSITORY) $(CI_PATH);
+-include $(MAKEFILE)
+
+# Makefile.main::dependencies -> Makefile.main::$(DEPENDENCIES) -> this::dependencies
+dependencies: | front-dependencies back-dependencies exit
+
+# Makefile.main::test -> this::test
+test: front-test
+
+# this::build -> Makefile.main::build -> Makefile.main::$(COMMANDS)
+build: front-build back-build
+	@echo
```

Comment:
could you please explain why do we need echo here?

Reply:
because it is needed to run this target before the `Makefile.main::build` one.
If you drop the recipe inside the target, it is run following the declaration order, what would be wrong in this case.

Reply:
wah. could you please add a comment about it. I see
> this::build -> Makefile.main::build -> Makefile.main::$(COMMANDS)
comment, but I didn't get it. Your explanation is much better.